SMLHelper/Patchers/LanguagePatcher.cs

```diff
@@ -112,13 +110,7 @@ private static void ReadOverrideCustomLines()
                     if (string.IsNullOrEmpty(line))
                         continue; // Skip empty lines
 
-                    string[] split = line.Split(new[] { KeyValueSeparator }, StringSplitOptions.RemoveEmptyEntries);
-
-                    if (split.Length != 2)
```

Comment:
The file being read is one that was manually edited.  
If we remove this safety check here, then we risk throwing an IndexOutOfRangeException when we do something like this:  
 ```
split[1]
```
Let's leave the safety checks in.